DataFormats/HcalRecHit/interface/HBHEChannelInfo.h

```diff
@@ -195,6 +206,11 @@ class HBHEChannelInfo
     // fC to GeV conversion factor (can depend on CAPID)
     double gain_[MAXSAMPLES];
 
+    // needed for the dark current
+    double darkCurrent_[MAXSAMPLES];
+    double fcByPE_[MAXSAMPLES];
+    double lambda_[MAXSAMPLES];
+
```

Comment:
@igv4321 
ok I can move to scalar, same should be true for these, shall I move all ?
         pedestal_[ts] = ped; pedestalWidth_[ts] = pedWidth; gain_[ts] = g;